### DISCUSSION TITLE
Changedetection - WebSocket error: connect ECONNREFUSED 127.0.0.1:3000

### DISCUSSION THREAD

**Discussion by mx4k** [tteck/Proxmox#2196] [Help Needed]
Hi, when I try to use the Playwright engine in changedetection, I get this error:

`WebSocket error: connect ECONNREFUSED 127.0.0.1:3000`

I think it's already mentioned here https://github.com/tteck/Proxmox/discussions/2070 but there is no answer too.

Any ideas?

**Comment by ViPeR5000** [comment]
Change Setttings .

Fetch Method
 Basic fast Plaintext/HTTP Client

**Comment by delicon** [comment]
Any news about this?
I'm still getting this error from time to time.
I'm solving it by restarting the container, but that's not a solution.

Just opened the CD dashboard and got all the websites with the same error:
`Exception: BrowserType.connect_over_cdp: WebSocket error: connect ECONNREFUSED 127.0.0.1:3000 Call log: <ws connecting> ws://127.0.0.1:3000/ - <ws error> ws://127.0.0.1:3000/ error connect ECONNREFUSED 127.0.0.1:3000 - <ws connect error> ws://127.0.0.1:3000/ connect ECONNREFUSED 127.0.0.1:3000 - <ws disconnected> ws://127.0.0.1:3000/ code=1006 reason=`

**Comment by MickLesk** [comment]
i think its an issue with browserless. after getting this error, you see following in the browserless.service:

May 24 15:37:06 changedetection browserless[377]:   <rejected> Error: Couldn't load route "/chrome/content" due to missing browser binary for "ChromeCDP"
May 24 15:37:06 changedetection browserless[377]:       at file:///opt/browserless/build/browserless.js:202:23
May 24 15:37:06 changedetection browserless[377]:       at Array.forEach (<anonymous>)
May 24 15:37:06 changedetection browserless[377]:       at Browserless.start (file:///opt/browserless/build/browserless.js:197:19)
May 24 15:37:06 changedetection browserless[377]: } reason: Error: Couldn't load route "/chrome/content" due to missing browser binary for "ChromeCDP"
May 24 15:37:06 changedetection browserless[377]:     at file:///opt/browserless/build/browserless.js:202:23
May 24 15:37:06 changedetection browserless[377]:     at Array.forEach (<anonymous>)
May 24 15:37:06 changedetection browserless[377]:     at Browserless.start (file:///opt/browserless/build/browserless.js:197:19)
May 24 15:37:06 changedetection browserless[377]: 2024-05-24T13:37:06.378Z browserless.io:index.js:info  Process is finished, exiting

And indeed, the whole chrome part is missing
